proto/cosmos/protocolpool/v1/budget.proto

```diff
@@ -0,0 +1,45 @@
+// Since: cosmos-sdk 0.50
+syntax = "proto3";
+package cosmos.protocolpool.v1;
+
+option go_package = "cosmossdk.io/x/protocolpool/types";
+
+import "gogoproto/gogo.proto";
+import "cosmos/base/v1beta1/coin.proto";
+import "cosmos_proto/cosmos.proto";
+import "cosmos/msg/v1/msg.proto";
+
+// Budget defines budget proposal type.
+//
+// Since: cosmos-sdk 0.50
+message MsgBudgetProposal {
```

Comment:
Usually msgs are in a `tx.proto` in the module. Same for queries in a `query.proto`
Could you follow that here too?